<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SparcCurationHelperWidget</class>
 <widget class="QWidget" name="SparcCurationHelperWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sparc Curation Helper</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="scaffold_annotation_tab">
      <attribute name="title">
       <string>Scaffold Annotation</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Annotated Scaffold Files:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2">
           <widget class="QLabel" name="scaffold_annotation_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QListView" name="scaffold_annotations_listView"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="thumbnail_preview_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Thumbnail</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="title">
          <string>Errors:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="1">
             <widget class="QPushButton" name="annotate_scaffold_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Annotate Scaffold</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="fixAllErrors_btn">
              <property name="text">
               <string>Fix All Errors</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Don't Push</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="fixError_btn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Fix Error</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QListView" name="errors_listView"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Files detected as Scaffold:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QListView" name="scaffold_metadata_listView"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="scaffold_metadata_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="plot_annotation_tab">
      <attribute name="title">
       <string>Plot Annotation</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButtonDone">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
